roles/mig_controller_deploy/tasks/deploy.yml

```diff
@@ -1,3 +1,21 @@
+- block:
+  - name: Update with mig controller custom image
+    lineinfile:
+      path: "{{ mig_controller_location }}/config/releases/latest/manifest.yaml"
+      regexp: '^(.*?)image:'
```

Comment:
You can probably set this as `'^( *?)image:'` to be more specific on indentation.

Reply:
Yep I wanted to ensure to catch everything rather than the yaml indentation, either way I think is good enough...